manifests/init.pp

```diff
@@ -57,7 +57,7 @@
   $opsgenie_tip               = 'is Ops genie up?',
   $pagerduty_runbook          = 'http://y/rb-pagerduty',
   $pagerduty_tip              = 'is PD up? https://events.pagerduty.com?',
-  $region                     = $::datacenter,
+  $environment                = $::datacenter,
```

Comment:
You only need to add environment but its default should be "" - so that by default it's the old behavior.

The key difference is you want this component of incident_key to be based on something from handler settings, while we at yelp want it to come from event json.